features/FEATURE_LWIP/lwip-interface/lwip_stack.c

```diff
@@ -370,19 +373,31 @@ int lwip_bringup(bool dhcp, const char *ip, const char *netmask, const char *gw)
         sys_arch_sem_wait(&lwip_tcpip_inited, 0);
 
         memset(&lwip_netif, 0, sizeof lwip_netif);
+#if DEVICE_EMAC
```

Comment:
Getting a bit nested ifdeffy - possibly better to match existing style and just ifdef around the relevant parameters? Or ifdef with a preload?